cmake/gtav-classes.cmake

```diff
@@ -2,8 +2,8 @@ include(FetchContent)
 
 FetchContent_Declare(
     gtav_classes
-    GIT_REPOSITORY https://github.com/Yimura/GTAV-Classes.git
-    GIT_TAG        68551ec2e5b09d6fe629abb7726148127d9f9e6f
+    GIT_REPOSITORY https://github.com/maybegreat48/GTAV-Classes.git
```

Comment:
Should this be changed back to Yimura's repo?

Reply:
Yim will do it when he goes to merge the changes.